spec/ics-1-ics-standard/README.md

```diff
@@ -0,0 +1,152 @@
+---
+ics: 1
+title: ICS Specification Standard
+stage: draft
+category: meta
+author: Christopher Goes <cwgoes@tendermint.com>
+created: 2019-02-12
+modified: 2019-03-04
+---
+
+## What is an ICS?
+
+An inter-chain standard (ICS) is a design document describing a particular protocol,
+standard, or feature expected to be of use to the Cosmos ecosystem.
+An ICS should list the desiderata of the standard, explain the design rationale, and
```

Comment:
Is `desiderata` contained in simple english :wink:

Reply:
No, mea culpa - replaced with "desired properties"